src/views/api-view.js

```diff
@@ -10,6 +10,9 @@
 
```

Comment:
suggestion for adding hint to use latest version: 

[api-view.txt](https://github.com/FACT-Finder-Web-Components/documentation/files/2552783/api-view.txt)